framework/components/ecl-forms/ecl-forms-checkboxes/ecl-checkboxes.scss

```diff
@@ -4,54 +4,49 @@
  */
 
 // checkbox related variables
-$form-checkbox-icon-size: 1.5em;
+$ecl-checkbox-icon-size: 1.5em;
 
 .ecl-checkbox {
-  opacity: 0;
-
-  // Checked state.
-  &:checked + .ecl-checkbox__label::before {
-    background-image: url('../images/svg/checkbox-checked--blue.svg');
-  }
-
-  // Focus state, not checked.
-  &:focus + .ecl-checkbox__label::before {
-    border-color: map-get($ecl-colors, 'yellow-100');
-  }
-
-  &[disabled] + .ecl-checkbox__label,
-  &[readonly] + .ecl-checkbox__label {
-    cursor: not-allowed;
-  }
+  display: block;
+  font-size: 1rem;
+  margin: 0;
+  padding: map-get($ecl-spacing, 'xxxs') 0;
```

Comment:
Quite a difference. I checked spec pages (even [old such](https://webgate.ec.europa.eu/CITnet/confluence/display/NEXTEUROPA/Checkbox)) and didn't find any specific info about this, so I hope it's ok.